webapp/components/search_bar.jsx

```diff
@@ -270,15 +274,6 @@ export default class SearchBar extends React.Component {
                 >
                     <span className='fa fa-angle-left'/>
                 </div>
-                <span
-                    className='search__clear'
-                    onClick={this.clearFocus}
-                >
-                    <FormattedMessage
-                        id='search_bar.cancel'
```

Comment:
is this string use elsewhere? if not can you please remove it from `webapp/i18n/en.json`?